packages/@ember/-internals/metal/lib/property_get.ts

```diff
@@ -125,15 +126,25 @@ export function _getProp(obj: object, keyName: string) {
       } else {
         value = (obj as MaybeHasUnknownProperty).unknownProperty!(keyName);
       }
-    }
 
-    if (isTracking()) {
-      consumeTag(tagFor(obj, keyName));
+      if (isTracking()) {
+        consumeTag(tagForProperty(obj, keyName));
```

Comment:
Moving to `tagForProperty` here seems a little bit unrelated to a glimmer-vm update, can you explain the motivation (and point out the previously failing test case that this resolves)?

Reply:
Ah, I committed this by mistake. My bad.